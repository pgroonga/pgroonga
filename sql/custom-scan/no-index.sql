CREATE TABLE ids (
  id integer
);

INSERT INTO ids VALUES (1);
INSERT INTO ids VALUES (2);
INSERT INTO ids VALUES (3);

SET pgroonga.enable_custom_scan = on;

EXPLAIN (COSTS OFF)
SELECT id
  FROM ids
 WHERE id = 2;

SELECT id
  FROM ids
 WHERE id = 2;

DROP TABLE ids;
