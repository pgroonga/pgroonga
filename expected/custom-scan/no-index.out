CREATE TABLE ids (
  id integer
);
INSERT INTO ids VALUES (1);
INSERT INTO ids VALUES (2);
INSERT INTO ids VALUES (3);
SET pgroonga.enable_custom_scan = on;
EXPLAIN (COSTS OFF)
SELECT id
  FROM ids
 WHERE id = 2;
     QUERY PLAN     
--------------------
 Seq Scan on ids
   Filter: (id = 2)
(2 rows)

SELECT id
  FROM ids
 WHERE id = 2;
 id 
----
  2
(1 row)

DROP TABLE ids;
